### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by jedireza**
Review request: :pager: @hueniverse @arb @Marsup

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -31,22 +25,22 @@ describe('register()', function () {
...
-        var server = new Hapi.Server();
+        const server = new Hapi.Server();
         server.connection();
-        server.register([Vision, VisionaryPlugin], function (err) {
+        server.register([Vision, VisionaryPlugin], (err) => {
 
             expect(err).to.not.exist();
 
-            var handler = function (request, reply) {
+            const handler = (request, reply) => {
```
Comment (Marsup) [inline test/index.js]: Can probably skip the braces here on these cases.

Comment (jedireza) [inline test/index.js]: I assume by braces you mean the parentheses. I like the structure they give. At the same time, I'd rather be unopinionated about this kind of stuff, so I checked the [styleguide](http://hapijs.com/styleguide) and noticed it's not updated for es6 syntax. Is it possible for the linter in `lab` to call out the right style in these cases?

Comment (Marsup) [inline test/index.js]: No, I mean `const handler = (request, reply) => reply.view('test', { message: 'hi' });`

Comment (jedireza) [inline test/index.js]: I updated these. Although I think we should mention this kind of stuff in the style guide eventually.

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,5 @@
 sudo: false
 language: node_js
 node_js:
-  - 0.10
-  - 0.12
-  - iojs
-
+  - "4"
+  - "5"
```
Comment (hueniverse) [inline .travis.yml]: 4.0, 4, 5
no quotes

**Comment by hueniverse** [comment]
Looks good.

**Comment by jedireza** [comment]
Thanks for the reviews.

**Comment by hueniverse** [comment]
Label, assignment...

**Pull request merged**